modes/chicken_fox.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Chicken-Fox-Grain",
+    "characters": ["Farmer", "Chicken", "Fox", "Grain"],
+    "boat_capacity": 2,
+    "rules": [
+      { "not_together": ["Chicken", "Grain"] },
+      { "not_together": ["Chicken", "Fox"] }
+    ]
+  }
+  
```

Comment:
Would be nice to remove the No new line at EOF, you can do so with `echo >> modes/chicken_fox.json`.